source/_patterns/molecules/blocks/hero/hero.twig

```diff
@@ -1,5 +1,12 @@
+{% set url_attributes = url_attributes ? url_attributes : create_attribute() %}
```

Comment:
This one doesn't need to be here, since the link is coming through the breadcrumbs.